suite-common/wallet-utils/src/__tests__/fiatRatesUtils.test.ts

```diff
@@ -0,0 +1,28 @@
+import { TokenAddress } from '@suite-common/wallet-types';
+
+import { getFiatRateKey, getFiatRateKeyFromTicker } from '../fiatRatesUtils';
+
+describe('fiat rates utils', () => {
+    it('formats fiat rate key', () => {
+        expect(getFiatRateKey('eth', 'usd')).toMatch('eth-usd');
+        expect(
+            getFiatRateKey(
+                'eth',
+                'usd',
+                '0x6b175474e89094c44da98b954eedeac495271d0f' as TokenAddress,
+            ),
+        ).toMatch('eth-0x6b175474e89094c44da98b954eedeac495271d0f-usd');
```

Comment:
[nitpic] I know this is maybe an absolute nitpick, but I like to suggest to not to add idententions in the code in general. IMO it reduces readability of the code.

I suggest to do it like:
```ts
it('formats fiat rate key', () => {
    expect(getFiatRateKey('eth', 'usd')).toMatch('eth-usd');

    const result = getFiatRateKey(
        'eth',
        'usd',
        '0x6b175474e89094c44da98b954eedeac495271d0f' as TokenAddress,
    );

    expect(result).toMatch('eth-0x6b175474e89094c44da98b954eedeac495271d0f-usd');
});
```